### PR TITLE
Time populate modal library

### PR DESCRIPTION
**Pull request by jun-w-huang**
1. Start and end time now populate correctly on profile page.
2. Refactored ConnectModal to use Headless' Dialog component

### CODE REVIEW

File: src/components/ConnectModal.tsx [hunk 1]
```diff
@@ -10,42 +11,56 @@ interface ConnectModalProps {
...
-
-      <div className="flex justify-center space-x-7">
-        <button
-          onClick={() => props.closeModal()}
-          className="w-full p-1 text-red-700 bg-slate-50 border-2 border-red-700 rounded-md"
-        >
-          Cancel
-        </button>
+const ConnectModal = (props: ConnectModalProps): JSX.Element => {
+  let [isOpen, setIsOpen] = useState(true);
```
Comment (lukejianu) [inline src/components/ConnectModal.tsx:15]: const

Comment (jun-w-huang) [reply src/components/ConnectModal.tsx]: 🙄

**Comment by lukejianu** [comment]
tried to pull your branch and test it locally but couldn't login

**Comment by jun-w-huang** [comment]
skill issue

**Comment by devashish9** [comment]
@jun-w-huang, did you to replicate @lukejianu's mentioned error? What happened?

**Review by devashish9 — COMMENTED** [review]
Looks good, let's just make sure this isn't breaking more things as @lukejianu potentially pointed out. Once you run this locally during standup and show me this isn't breaking, I'll give you merge approval

**Comment by DavidOB1** [comment]
Yo, listen up, let me tell you 'bout a change
That Jun made to our code, it's kinda strange
He fixed the onboarding page, now it's all good
Start and end time fields, they're working like they should
But hold up, we can't approve it just yet
We need to make sure there ain't no bugs to fret
Jun, my man, you did a great job
But we gotta test it out before we can applaud
And there's more, Jun went and did it again
Refactored ConnectModal, it's the new trend
Used Headless' Dialog component, it's so sweet
Makes our code cleaner and more complete
So, Jun, we're grateful for the work you've done
But we gotta make sure it's bug-free, it's number one
We'll test it out, put it through its paces
Make sure it's solid, without any traces
So that's it, Jun, you've made a great change
But we gotta make sure it's in our range
We'll test it out, make sure it's legit
Then we'll approve it and give you a big tip!

**Review by devashish9 — APPROVED** [review]

**Pull request merged**